database.hh

```diff
@@ -708,7 +708,7 @@ public:
             streamed_mutation::forwarding fwd = streamed_mutation::forwarding::no,
             mutation_reader::forwarding fwd_mr = mutation_reader::forwarding::yes) const;
     flat_mutation_reader make_reader_excluding_sstable(schema_ptr schema,
```

Comment:
The name should now be plural, no?

Reply:
yeah, I think I had fixed up all occurrences but let this one slip. Thanks :vampire: